static/app/components/events/interfaces/nativeFrame.tsx

```diff
@@ -336,158 +323,122 @@ function NativeFrame({
+const StatusCell = styled(Cell)``;
 
-const PackageCell = styled(Cell)`
-  color: ${p => p.theme.subText};
-  @media (max-width: ${p => p.theme.breakpoints.small}) {
-    grid-column: 2/2;
-    grid-row: 1/1;
-    display: grid;
-    grid-template-columns: 1fr;
-    grid-template-rows: repeat(2, auto);
-  }
-`;
+const PackageCell = styled(Cell)``;
 
 const AddressCell = styled(Cell)`
+  ${p => p.onClick && `cursor: pointer`};
+  ${p => p.onClick && `color:` + p.theme.linkColor};
+`;
+
+const FunctionNameCell = styled(Cell)`
   font-family: ${p => p.theme.text.familyMono};
+  word-break: break-all;
+
   @media (max-width: ${p => p.theme.breakpoints.small}) {
-    grid-column: 3/3;
-    grid-row: 1/1;
+    grid-column: 2/6;
   }
 `;
 
 const GroupingCell = styled(Cell)`
   @media (max-width: ${p => p.theme.breakpoints.small}) {
-    grid-column: 1/1;
-    grid-row: 2/2;
+    grid-row: 2/3;
   }
 `;
 
-const FunctionNameCell = styled(Cell)`
-  color: ${p => p.theme.textColor};
+const TypeCell = styled(Cell)`
   @media (max-width: ${p => p.theme.breakpoints.small}) {
-    grid-column: 2/-1;
-    grid-row: 2/2;
+    grid-column: 5/6;
+    grid-row: 1/2;
   }
 `;
 
-const ExpandCell = styled(Cell)``;
+const ExpandCell = styled(Cell)`
+  @media (max-width: ${p => p.theme.breakpoints.small}) {
+    grid-column: 6/7;
+    grid-row: 1/2;
+  }
+`;
 
-const RegistersCell = styled('div')`
-  grid-column: 1/-1;
-  margin-left: -${space(0.5)};
-  margin-right: -${space(0.5)};
-  margin-bottom: -${space(0.5)};
-  cursor: default;
+const ToggleButton = styled(Button)`
+  width: 16px;
+  height: 16px;
 `;
 
 const Registers = styled(Context)`
+  border-bottom: 1px solid ${p => p.theme.border};
   padding: 0;
   margin: 0;
 `;
 
-const ToggleButton = styled(Button)`
-  width: 16px;
-  height: 16px;
-`;
-
-const Package = styled('span')<{color?: ColorOrAlias}>`
-  border-bottom: 1px dashed ${p => p.theme.border};
-  ${p => p.color && `color: ${p.theme[p.color]}`};
-  ${p => p.onClick && `cursor: pointer;`}
+const PackageNote = styled('div')`
+  color: ${p => p.theme.subText};
+  font-size: ${p => p.theme.fontSizeExtraSmall};
 `;
 
-const FunctionName = styled('div')`
-  color: ${p => p.theme.headingColor};
-  margin-right: ${space(1)};
+const Package = styled('span')`
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  width: 100%;
 `;
 
 const FileName = styled('span')`
   color: ${p => p.theme.subText};
   border-bottom: 1px dashed ${p => p.theme.border};
 `;
 
-const PackageStatusButton = styled(Button)`
-  padding: 0;
-  border: none;
-`;
+const FrameRow = styled('li')``;
 
-const GridRow = styled('li')<{expandable: boolean; expanded: boolean; inApp: boolean}>`
+const RowHeader = styled('span')<{expandable: boolean; expanded: boolean}>`
+  display: grid;
+  grid-template-columns: repeat(2, auto) 1fr repeat(2, auto);
+  grid-template-rows: repeat(2, auto);
+  align-items: center;
+  align-content: center;
+  grid-column-gap: ${space(1)};
```

Comment:
i think this is the old name
```suggestion
  column-gap: ${space(1)};
```